problems/my_reject.rb

```diff
@@ -1,3 +1,6 @@
+# Write an `Array#my_reject(&prc)` method.
+# This method should return a new array excluding all the elements in the original array that satisfy the proc.
+# Example: `[1,2,3].my_reject {|n| n.even?}` => [1,3]
```

Comment:
So for this one I want to disallow both `select` and `reject`. However, unlike some of the others ones weird stuff will happen with rspec if we say `expect_any_instance_of(Array).not_to receive(:select)` so here's the workaround I came up with and implemented for A01:

```
let(:arr) { [1,2,3] }
  before(:each) do
    expect(arr).not_to receive(:select)
    expect(arr).not_to receive(:dup)
    expect_any_instance_of(Array).not_to receive(:select!)
    expect_any_instance_of(Array).not_to receive(:reject)
    expect_any_instance_of(Array).not_to receive(:reject!)
    arr.my_select { |el| el }
  end
```

The idea is we expect the instance itself not to receive `select` - but students could easily get around this by duping the array, so we disallow the `dup` method as well on that instance. It's not perfect but I think it's good enough and about all we can do. Unless you think of something else, then let me know!